changelog/2926.trivial.rst

```diff
@@ -0,0 +1,9 @@
+Users can now use fewer keywords in our `~sunpy.net.HelioviewerClient` to access the available sources. Either by `observatory` and `measurement` or `instrument` and `measurement` as this much information is enough to get the source ID for most of the cases.
+
+For example, if we want to use `~sunpy.net.helioviewer.HelioviewerClient.download_jp2` you can now do the  following:
+
+1. filepath = hv.download_jp2('2012/07/03 14:30:00', observatory='SDO', instrument='HMI', detector=None, measurement='continuum')
+2. filepath = hv.download_jp2('2012/07/03 14:30:00', observatory='SDO', measurement='continuum')
+3. filepath = hv.download_jp2('2012/07/03 14:30:00', instrument='HMI', measurement='continuum')
```

Comment:
We had a discussion on this and in hindsight, this should be in the documentation string for the functions to tell people they can do this instead of the changelog.

Reply:
oh ok I will do that

Reply:
We need to add some comments to each doc string that uses sourceID, telling users that you can do this.

Reply:
Like telling the user that in place of source ID we can use the partial key to retrieve the source ID?

Reply:
Yes